kasa/tests/test_emeter.py

```diff
@@ -39,12 +40,13 @@ async def test_no_emeter(dev):
 
     with pytest.raises(SmartDeviceException):
         await dev.get_emeter_realtime()
-    with pytest.raises(SmartDeviceException):
-        await dev.get_emeter_daily()
-    with pytest.raises(SmartDeviceException):
-        await dev.get_emeter_monthly()
-    with pytest.raises(SmartDeviceException):
-        await dev.erase_emeter_stats()
+    if isinstance(dev, IotDevice):
```

Comment:
Please add a comment why some are tested against all and these only against iotdevices.

Reply:
Done